lib/money/money.rb

```diff
@@ -144,6 +147,8 @@ def self.default_currency
       Money::Currency.new(@default_currency.call)
     elsif @default_currency
       Money::Currency.new(@default_currency)
+    else
+      raise DefaultCurrencyNotSetOrPassed
```

Comment:
Btw, maybe raising it in the getter method isn't the best approach since it'll raise when a developer is checking the current value `Money. default_currency`

Reply:
Good point. I will review and find the best place for this. thank you @antstorm